README.md

```diff
@@ -22,14 +16,29 @@ The easiest deployment mechanism is to install [Chocolatey](https://chocolatey.o
 Then run `beacon` and observe the command-line arguments.
 
 ## Example usage
+### Azure Devops
+Using a public Azure Devops project and running continuously:
+
+    beacon azuredevops --url https://dev.azure.com/my-public-project --project MyProject --definitionid 1
+
+Or, using a personal access token (e.g. 1234567890ab, see [here](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page) for how to obtain a token) "for a private project and a custom branch name "branch/my":
 
+    beacon azuredevops --url https://dev.azure.com/my-public-project --project MyProject --definitionid 1 --token 1234567890ab -b -b refs/heads/branch/my
```

Comment:
Does it support multiple definition IDs?

Reply:
I see `-b` twice. Is that a typo?

Reply:
> Does it support multiple definition IDs?

No

Reply:
At my client, we have usually have about 10 builds in different folders that we monitor.

Reply:
I could add support for multiple pipelines (i.e. definition ids) per Azure DevOps project, e.g. https://dev.azure.com/avivasolutions-public/Nyxie/_build has 2 pipelines that could be monitored with the same build light.

What do you think?

Reply:
That would be very useful IMO and consistent with the TeamCity implementation.

Reply:
Implemented: https://github.com/dennisdoomen/Beacon/pull/33/commits/9c23333d5fd7ee37687f6e9097a6b558b0966a27